### PR TITLE
Add UIN to Gradebook #1304

### PR DESCRIPTION
**Pull request by rongni**
added UIN to instructor's gradebook

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4920,8 +4920,25 @@
...
+                "speech-rule-engine": {
+                    "version": "3.0.0-beta.8",
+                    "resolved": "https://registry.npmjs.org/speech-rule-engine/-/speech-rule-engine-3.0.0-beta.8.tgz",
+                    "integrity": "sha512-gnNap93Xrz5ngIqa/BOaJGatC64B7dqDrXK/pr4yAaZAcW6tKVO5PwaTvw9QqVab86ad5scpA39Nxwhlb11tTg==",
+                    "requires": {
+                        "commander": "*",
+                        "wicked-good-xpath": "*",
+                        "xmldom-sre": "^0.1.31"
+                    }
+                }
```
Comment (mwest1066) [inline package-lock.json]: Why do we have changes to `package-lock.json`? Are these needed?

File: pages/instructorGradebook/instructorGradebook.sql [hunk 2]
```diff
@@ -35,22 +35,22 @@ user_ids AS (
     (SELECT user_id FROM enrollments WHERE course_instance_id = $course_instance_id)
 ),
 course_users AS (
-    SELECT u.user_id,u.uid,u.name AS user_name,coalesce(e.role, 'None'::enum_role) AS role
+    SELECT u.user_id,u.uid,u.uin, u.name AS user_name,coalesce(e.role, 'None'::enum_role) AS role
```
Comment (mwest1066) [inline pages/instructorGradebook/instructorGradebook.sql]: Looks like an extra space was added after `u.uin`?

File: package-lock.json (hunk 1 above)
Comment (rongni) [reply package-lock.json]: sorry, I think we changed this file by accident, can you just remove the changes we did for this file? thanks !!!

Comment (mwest1066) [reply package-lock.json]: Can you revert these, as PR author? You can use git commands to revert them.

**Comment by RDT04** [comment]
I will be working on this project.

**Comment by nicknytko** [comment]
Continued in #2153

**Pull request closed without merging**